templates/projects/view.tmpl

```diff
@@ -64,7 +64,7 @@
 </div>
 
 <div id="project-board">
-	<div class="board {{if .CanWriteProjects}}sortable{{end}}">
+	<div class="board {{if .CanWriteProjects}}sortable{{end}}"{{if .CanWriteProjects}} data-url="{{$.Link}}/move"{{end}}>
```

Comment:
`{{if .CanWriteProjects}}` twice?

Reply:
The first one to add a new class, the second one to add a new attribute. I don't think mixing them is a good idea for readability.